app/views/users/memberList.scala.html

```diff
@@ -60,7 +60,8 @@
                                 aria-label="@messages("general.close")">
                             <span aria-hidden="true">&times;</span>
                         </button>
-                        <h4 class="modal-title" id="label-user-delete">@messages("project.removeMember")</h4>
+                        <h4 class="modal-title" id="label-use
+                        r-delete">@messages("project.removeMember")</h4>
```

Comment:
make this one line again

Reply:
Whoops, don't know how that typo got in there.

Reply:
Typo fixed.